tools/src/compileProtos.ts

```diff
@@ -246,7 +246,7 @@ async function compileProtos(
       '-p',
       'protos',
       '-p',
-      path.join(__dirname, '..', '..', '..', 'google-gax', 'build', 'protos'),
+      path.join('node_modules', 'google-gax', 'build', 'protos'),
```

Comment:
Let's do it properly: `path.join(path.dirname(require.resolve('google-gax')), '..', 'protos')`

`require.resolve('google-gax')` will point to `build/src/index.js` so `path.dirname` will give `src`, one level up is `build` and then `protos` - should work